steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,234 @@ The installation has already been performed on an OpenShift cluster as per the i
+- Fill in the username and password fields with the `username` and `password` values from the EnMasse binding secret.
+- Choose `Disable` for the Check Certificates dropdown
+- Click Validate 
+- Click the `Next` button at the top right
+- Add a connection name ```enmasse```
+- Click the `Create` button at the top right
 
-# Setup HTTP connection to the messaging booster in Fuse Ignite
-- Open the Fuse ignite console and open the connections again. 
-- Click create connection
-- Find the HTTP connection
-- Set the base URL as the url exposed from the messaging front end application
-- Click validate to ensure Fuse Ignite can reach the endpoint
-- Click next 
-- Give the connection a name (booster)
-- Click create
+# Setup HTTP connection to the CRUD REST booster in Fuse Ignite
+
+- Stay in the Fuse Ignite console for this section
+- Select `Connections` and `Create connection`
+- Find the `HTTP Connector` and select it. 
+- Set the base URL as the url exposed from the CRUD REST booster application that we created earlier (You can get this using `oc get route spring-boot-rest-http-crud -n eval`)
+- Click `Validate` to ensure Fuse Ignite can reach the endpoint
+- Click `Next` 
+- Give the connection a name (crud booster)
+- Click `Create`
 
 # Creating the integration
 
-- Open the Fuse Ignite console
-- Click on integrations
-- Click create integration
+- Stay in the Fuse Ignite console for this section
+- Click on `Integrations` and `Create Integration`
 - Click on the Enmaase connection
-- Click on the ``` Subscribe for messages ``` option
-- Under Destination Name add ``` work-queue/worker-updates ```
-- Choose Queue as the Destination Type
-- Click Next
-- Set the Specify Output Data Type as ``` Type specification not required ``` this is the default by clicking done
-- On the ``` Choose a Finish Connection ``` screen select the messaging booster connection you created
-- Choose ```Invoke URL ``` 
-- In the URL Path set the path to ```/api/data```
-- In the method select ```GET``` and click next
-- Again under Specify Input Data Type leave it as ``` Type specification not required ``` and click done
-- Click publish at the top right
-- Choose the integration name (steel thread)
+- Click on the ```Subscribe for messages``` option
+- Under Destination Name add ```work-queue/requests ```
+- Choose `Queue` as the Destination Type
+- Click `Next`
+- Set the `Select Type` as ```JSON Schema``` 
+- Add the following schema to the `Definition` field & click `Done`
+```
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"type": {
+			"type": "string"
+		},
+		"stock": {
+			"type": "string"
+		}
+	}
+}
+```
+- On the ```Choose a Finish Connection``` screen select the `crud booster` connection you created
+- Choose ```Invoke URL``` 
+- In the URL Path set the path to ```/api/fruits```
+- In the method select ```POST``` and click next
+- Again under `Select Type` set it to ```JSON Schema``` 
```

Comment:
Need to click 'Next' button before you can update the Select Type and JSON schema